classnotfound when starting dev container

I get the following error when starting the devcontainer
```
Failed to start quarkus: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor#startContainers threw an exception: java.lang.RuntimeException: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.joda.JodaModule could not be instantiated
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainers(DevServicesOpenFGAProcessor.java:101)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:829)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.joda.JodaModule could not be instantiated
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:582)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:804)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:722)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1395)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1105)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1089)
	at com.fasterxml.jackson.databind.ObjectMapper.findAndRegisterModules(ObjectMapper.java:1139)
	at io.quarkiverse.openfga.deployment.DevServicesStoreInitializer.<clinit>(DevServicesStoreInitializer.java:21)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.lambda$startContainer$5(DevServicesOpenFGAProcessor.java:168)
	at java.base/java.util.Optional.orElseGet(Optional.java:369)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainer(DevServicesOpenFGAProcessor.java:243)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainers(DevServicesOpenFGAProcessor.java:85)
	... 11 more
Caused by: java.lang.NoClassDefFoundError: org/joda/time/DateTime
	at com.fasterxml.jackson.datatype.joda.JodaModule.<init>(JodaModule.java:20)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:780)
	... 21 more
Caused by: java.lang.ClassNotFoundException: org.joda.time.DateTime
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:490)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:440)
	... 27 more

	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:330)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:252)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:60)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.firstStart(IsolatedDevModeMain.java:87)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:448)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:60)
	at io.quarkus.bootstrap.app.CuratedApplication.runInCl(CuratedApplication.java:148)
	at io.quarkus.bootstrap.app.CuratedApplication.runInAugmentClassLoader(CuratedApplication.java:103)
	at io.quarkus.deployment.dev.DevModeMain.start(DevModeMain.java:131)
	at io.quarkus.deployment.dev.DevModeMain.main(DevModeMain.java:62)
Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor#startContainers threw an exception: java.lang.RuntimeException: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.joda.JodaModule could not be instantiated
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainers(DevServicesOpenFGAProcessor.java:101)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:829)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.joda.JodaModule could not be instantiated
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:582)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:804)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:722)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1395)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1105)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1089)
	at com.fasterxml.jackson.databind.ObjectMapper.findAndRegisterModules(ObjectMapper.java:1139)
	at io.quarkiverse.openfga.deployment.DevServicesStoreInitializer.<clinit>(DevServicesStoreInitializer.java:21)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.lambda$startContainer$5(DevServicesOpenFGAProcessor.java:168)
	at java.base/java.util.Optional.orElseGet(Optional.java:369)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainer(DevServicesOpenFGAProcessor.java:243)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainers(DevServicesOpenFGAProcessor.java:85)
	... 11 more
Caused by: java.lang.NoClassDefFoundError: org/joda/time/DateTime
	at com.fasterxml.jackson.datatype.joda.JodaModule.<init>(JodaModule.java:20)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingctorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:780)
	... 21 more
Caused by: java.lang.ClassNotFoundException: org.joda.time.DateTime
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:490)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:440)
	... 27 more

	at io.quarkus.builder.Execution.run(Execution.java:123)
	at io.quarkus.builder.BuildExecutionBuilder.execute(BuildExecutionBuilder.java:79)
	at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:158)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:328)
	... 9 more
Caused by: java.lang.RuntimeException: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.joda.JodaModule could not be instantiated
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainers(DevServicesOpenFGAProcessor.java:101)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:829)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: Provider com.fasterxml.jackson.datatype.joda.JodaModule could not be instantiated
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:582)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:804)
	at java.base/java.util.ServiceLoader$ProviderImpl.get(ServiceLoader.java:722)
	at java.base/java.util.ServiceLoader$3.next(ServiceLoader.java:1395)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1105)
	at com.fasterxml.jackson.databind.ObjectMapper.findModules(ObjectMapper.java:1089)
	at com.fasterxml.jackson.databind.ObjectMapper.findAndRegisterModules(ObjectMapper.java:1139)
	at io.quarkiverse.openfga.deployment.DevServicesStoreInitializer.<clinit>(DevServicesStoreInitializer.java:21)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.lambda$startContainer$5(DevServicesOpenFGAProcessor.java:168)
	at java.base/java.util.Optional.orElseGet(Optional.java:369)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainer(DevServicesOpenFGAProcessor.java:243)
	at io.quarkiverse.openfga.deployment.DevServicesOpenFGAProcessor.startContainers(DevServicesOpenFGAProcessor.java:85)
	... 11 more
Caused by: java.lang.NoClassDefFoundError: org/joda/time/DateTime
	at com.fasterxml.jackson.datatype.joda.JodaModule.<init>(JodaModule.java:20)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:780)
	... 21 more
Caused by: java.lang.ClassNotFoundException: org.joda.time.DateTime
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:490)
	at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:440)
	... 27 more
```

--- a/deployment/src/main/java/io/quarkiverse/openfga/deployment/DevServicesStoreInitializer.java
+++ b/deployment/src/main/java/io/quarkiverse/openfga/deployment/DevServicesStoreInitializer.java
@@ -10,6 +10,7 @@ import java.util.Optional;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import io.quarkiverse.openfga.client.api.API;
 import io.quarkiverse.openfga.client.model.Store;
 import io.quarkiverse.openfga.client.model.dto.CreateStoreRequest;
 import io.quarkiverse.openfga.client.model.dto.CreateStoreResponse;
@@ -18,7 +19,7 @@ import io.quarkiverse.openfga.client.model.dto.WriteAuthorizationModelResponse;
 
 public class DevServicesStoreInitializer {
 
-    private static final ObjectMapper objectMapper = new ObjectMapper().findAndRegisterModules();
+    private static final ObjectMapper objectMapper = API.createObjectMapper();
 
     private final HttpClient httpClient = HttpClient.newHttpClient();
     private final URI instanceURL;

--- a/runtime/pom.xml
+++ b/runtime/pom.xml
@@ -31,6 +31,22 @@
       <artifactId>arc</artifactId>
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jsr310</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jdk8</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-parameter-names</artifactId>
+    </dependency>
+    <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-smallrye-health</artifactId>
       <optional>true</optional>

--- a/runtime/src/main/java/io/quarkiverse/openfga/client/api/API.java
+++ b/runtime/src/main/java/io/quarkiverse/openfga/client/api/API.java
@@ -52,11 +52,7 @@ public class API {
     public API(WebClient webClient, Optional<Credentials> credentials) {
         this.webClient = webClient;
         this.credentials = credentials;
-        this.objectMapper = new ObjectMapper()
-                .registerModule(new JavaTimeModule())
-                .registerModule(new Jdk8Module())
-                .registerModule(new ParameterNamesModule())
-                .setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        this.objectMapper = createObjectMapper();
     }
 
     public void close() {
@@ -302,6 +298,14 @@ public class API {
                 .setURI(uriTemplate.expandToString(variables))
                 .setTraceOperation(format("FGA | %s", operationName.toUpperCase()));
         return webClient.request(method, options);
+    }
+
+    public static ObjectMapper createObjectMapper() {
+        return new ObjectMapper()
+                .registerModule(new JavaTimeModule())
+                .registerModule(new Jdk8Module())
+                .registerModule(new ParameterNamesModule())
+                .setSerializationInclusion(JsonInclude.Include.NON_NULL);
     }
 
     private static final String APPLICATION_JSON = "application/json";

